dataflow/lib/exporters.py

```diff
@@ -0,0 +1,80 @@
+from functools import wraps
```

Comment:
Maybe provide higher level functions, one for columns with key-value headers (text, csv) and another for structured hierarchies (hdf, json)?

Reply:
That's what this is... all csv will use "exports_text", and structured files are all different enough that they will need their own code.